packages/api/src/routes/medical/internal-hie.ts

```diff
@@ -42,4 +44,46 @@ router.get(
   })
 );
 
+/**
+ * GET /internal/hie/enabled
+ *
+ * Retrieves the customer status of enabled HIEs via the Feature Flags.
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ */
+router.get(
+  "/hie/enabled",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const response = await getHieEnabledFFStatus(cxId);
+    return res.status(httpStatus.OK).json(response);
+  })
+);
+
+/**
+ * POST /internal/hie/enabled
+ *
+ * Updates the customer status of enabled HIEs via the Feature Flags.
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ * @param req.query.cwEnabled - Whether to enabled CommonWell.
+ * @param req.query.cqEndabled - Whether to enabled CareQuality.
+ */
+router.post(
```

Comment:
since were updating should this be a PUT

Reply:
Done. Also moved to just internal as these aren't really "medical" endpoints in retrospect.